BibTeX.js

```diff
@@ -262,7 +263,14 @@ function setKeywordDelimRe( val, flags ) {
 function processField(item, field, value, rawValue) {
 	if(Zotero.Utilities.trim(value) == '') return null;
 	if(fieldMap[field]) {
-		item[fieldMap[field]] = value;
+		//map DOIs + Label to Extra for unsupported item types
```

Comment:
Unsupported fields will not be saved in Zotero such that there will never be two fields containing the DOI information. No?

Anyway, there is no big deal here. I just thought it might be easier to read (and maybe already work correctly in some future version of Zotero item types).

I will review your changes a little later, but I am sure everything is good.